readme.md

```diff
@@ -45,10 +45,33 @@ Type: `object`
 
 ##### deep
 
-Type: `boolean`<br>
+Type: `boolean | Function`<br>
 Default: `false`
 
-Recurse nested objects and objects in arrays.
+Recursing behaviour for nested objects and objects in arrays
+
+Deep Value can be boolean:
```

Comment:
Not sure why this is capitalized.

Reply:
You're right, this might confuse people. I hope the given WA is more clear now